dlp/src/main/java/dlp/snippets/JobsDelete.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dlp.snippets;
+
+// [START dlp_delete_job]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.DeleteDlpJobRequest;
+import com.google.privacy.dlp.v2.DlpJobName;
+
+import java.io.IOException;
+
+public class JobsDelete  {
+    public static void deleteJobs() throws IOException {
+        // TODO(developer): Replace these variables before running the sample.
+        String projectID = "your-project-id";
```

Comment:
nit: Do we use caps in other places? We should be consistent with values we expect users to replace.

Reply:
The existing reformatted snippets weren't consistent with this. I updated all of them to use lowercase letters in the last commit, because project ids tend to be lowercase, but I can also make them all uppercase if needed